src/stageros.cpp

```diff
@@ -327,23 +325,23 @@ StageNode::StageNode(int argc, char** argv, bool gui, const char* fname, bool us
 int
 StageNode::SubscribeModels()
 {
-    n_.setParam("/use_sim_time", true);
+//    n_->set_parameter(rclcpp::Parameter("/use_sim_time", true));
```

Comment:
I guess this is not really necessary as nodes are unable to read parameters defined in other nodes?

Reply:
Indeed, I removed it.

For reference: I checked [gazebo_ros_pkgs](https://github.com/ros-simulation/gazebo_ros_pkgs/tree/eloquent) for references to `use_sim_time` and didn't find any. If I understand correctly, the intention is to set `use_sim_time` through launch files, so the simulator doesn't have to.